product_contract/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Product Contract
+================
+
+This module adds support for products to be linked to contracts. It also adds
+functionality to automatically create a contract when a ``sale.order`` contains
+a product that implements a contract.
```

Comment:
Shouldn't you add a many2one in the sales order line to keep a reference to the created contract? Even an smart-button like the one that `project` adds to the order can be good.